src/main/kotlin/server/mcp/Task.kt

```diff
@@ -47,8 +49,8 @@ class Task(
 
 
     sealed interface Result {
-        data object Finished: Result
-        @JvmInline value class Suspend(val seconds: Int): Result
+        data class Finished(val v: Value?): Result
+        data class Suspend(val seconds: Int): Result
```

Comment:
There was very little actual optimization value in making these jvminline value classes to begin with, so I'm not doing it here anymore.